float32 pitch_rate
float32 roll_rate
float32 yaw_rate
float32 thrust
---
bool success
string message
